indico/htdocs/js/utils/misc.js

```diff
@@ -169,4 +171,33 @@
         value = value.trim();
         return value ? value.split(/\s+/).length : 0;
     };
+
+    function _doUpdateHtml(target, html, replace, highlight) {
+        var $target = $(target);
+        if (replace) {
+            var $html = $($.parseHTML(html, document, true));
+            $target.replaceWith($html);
+            $target = $html;
+        } else {
+            $target.html($.parseHTML(html, document, true));
+        }
+        $target.trigger('indico:htmlUpdated');
+        if (highlight) {
+            $target.effect('highlight', {color: '#5D95EA'});
```

Comment:
Since we can't easily extract it from the scss file I think I'd rather add it as a new palette.js file though instead of going through vars.js